Gems/ROS2/Code/Source/Clock/SimulationClock.cpp

```diff
@@ -36,8 +36,13 @@ namespace ROS2
         }
     }
 
+    int64_t SimulationClock::GetMedianExecutionTime() const {
+        return m_currentMedian;
+    }
```

Comment:
standards: brace placement

```
int64_t SimulationClock::GetMedianExecutionTime() const
{
        return m_currentMedian;
}
```

Reply:
Also please prefer `AZ::s64` to better handle weird `long`/`long long` differences on Windows/Linux (tagging @lumberyard-employee-dm, I believe this is preferred).